test/test_autotag.py

```diff
@@ -91,7 +93,10 @@ def test_current_metadata_likelies(self):
                  for i in range(5)]
         likelies, _ = match.current_metadata(items)
         for f in fields:
-            self.assertEqual(likelies[f], '%s_1' % f)
+            if type(items[0]._fields[f]) in (Integer, PaddedInt):
```

Comment:
Maybe a slightly easier way to do this would be `isinstance(likelies[f], int)`?

Reply:
yes, it looks simpler.